Makefile

```diff
@@ -40,18 +44,22 @@ deps:
 	go get -u github.com/goadesign/gorma
 
 generate: $(DESIGNS)
-	goagen bootstrap -d ${PACKAGE_NAME}/${DESIGNDIR}
-	goagen js -d ${PACKAGE_NAME}/${DESIGNDIR} -o assets/ --noexample
-	goagen gen -d ${PACKAGE_NAME}/${DESIGNDIR} --pkg-path=github.com/goadesign/gorma
-	go-bindata-assetfs -debug assets/...
-	godep get
+	$(GOPATH)/bin/goagen bootstrap -d ${PACKAGE_NAME}/${DESIGNDIR}
```

Comment:
I disagree, since IDE's will NOT pick up changes done in a command line window, but I can back out of this change if you insist.